libs/ledger-live-common/src/wallet-api/react.ts

```diff
@@ -350,24 +345,13 @@ export function useWalletAPIServer({
 
       return new Promise((resolve, reject) => {
         // handle no curencies selected case
-        const currencyIds = currencies.map(({ id }) => id);
-
-        let currencyList: CryptoOrTokenCurrency[] = [];
-        // if single currency available redirect to select account directly
-        if (currencyIds.length === 1) {
-          const currency = findCryptoCurrencyById(currencyIds[0]) || findTokenById(currencyIds[0]);
+        const currencyList = currencies.reduce<CryptoOrTokenCurrency[]>((prev, { id }) => {
+          const currency = findCryptoCurrencyById(id) || findTokenById(id);
           if (currency) {
-            currencyList = [currency];
+            prev.push(currency);
           }
-
-          if (!currencyList[0]) {
-            tracking.requestAccountFail(manifest);
```

Comment:
Was that error tracking useless (since its being removed)?

Reply:
Yep